docs/core/deploying/index.md

```diff
@@ -1,95 +1,120 @@
 
-## Self-contained deployments (SCD)
+```dotnet
+dotnet publish -r <RID>
+```
 
-For a self-contained deployment, you deploy your app and any required third-party dependencies along with the version of .NET Core that you used to build the app. Creating an SCD doesn't include the [native dependencies of .NET Core](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md) on various platforms, so these must be present before the app runs. For more information on version binding at runtime, see the article on [version binding in .NET Core](../versions/selection.md).
+### Advantages
 
-Starting with NET Core 2.1 SDK (version 2.1.300), .NET Core supports *patch version roll forward*. When you create a self-contained deployment, .NET Core tools automatically include the latest serviced runtime of the .NET Core version that your application targets. (The latest serviced runtime includes security patches and other bug fixes.) The serviced runtime does not have to be present on your build system; it is downloaded automatically from NuGet.org. For more information, including instructions on how to opt out of patch version roll forward, see [Self-contained deployment runtime roll forward](runtime-patch-selection.md).
+- **Control .NET Core version**\
+You control which version of .NET Core is deployed with your app. .NET Core can only be upgraded by releasing a new version of your app.
 
-FDD and SCD deployments use separate host executables, so you can sign a host executable for an SCD with your publisher signature.
+- **Platform-specific targeting**\
+Because you have to publish your app for each platform, you know where your app will run. If .NET Core introduces a new platform, users can't run your app on that platform until you release a version that is published that platform. You can test your app for compatibility problems before your users run your app on the new platform.
 
-### Why deploy a self-contained deployment?
+### Disadvantages
 
-Deploying a Self-contained deployment has two major advantages:
+- **Larger deployments**\
+Because your app includes the .NET Core runtime and all of your app dependencies, the hard drive space required is greater than a [runtime-dependent](#runtime-dependent) version.
 
-- You have sole control of the version of .NET Core that is deployed with your app. .NET Core can be serviced only by you.
+  > [!TIP]
+  > You can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core [*globalization invariant mode*](https://github.com/dotnet/runtime/blob/master/docs/design/features/globalization-invariant-mode.md). This forces your app to treat all cultures like the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture?displayProperty=nameWithType).
 
-- You can be assured that the target system can run your .NET Core app, since you're providing the version of .NET Core that it will run on.
+- **More space required across all apps**\
```

Comment:
This seems like a duplicate of the previous bullet.

Reply:
yes.. this was here in the original, I didn't like but kept it. I'll remove it though